### ISSUE TITLE
Fix color schemes

### ISSUE THREAD

**Issue by lgarron** [cubing/alg.cubing.net#23]
Right now, only BOY and Japanese work.
- [ ] Decide on a validation scheme.

**Comment by tussosedan** [comment]
Would you like help implementing this? I'd be happy to put some time and effort into it.

**Comment by renatoluz** [comment]
i believe it's about it...i'd like to change the views

i was trying to practice the cross...is there any feature? i'm learning to use. i'd be happy to use this!

**Comment by lgarron** [comment]
My goal is to get this working in v2, but unfortunately I can't offer a timeline for that. :-/

**Comment by tussosedan** [comment]
@lgarron that would be awesome, I'm a bit tired of running my own modified version locally on my devices heh

**Comment by yufanyufan** [comment]
Can we fix this by adding one built-in schema to the list?
    {id: "girl", name: "GIRL", type: "Color Scheme", scheme: "gorbwy", display: "Girl", custom: false},

I tested this locally and it works.

**Comment by jkortus** [comment]
for anyone still interested after all those years, I applied yufanyufan's fix and deployed the forked version on https://alg.cubing.cz. Please let me know if there are any problems, as I'm not an expert in this code or its logic at all :-). Forked repo with changes - https://github.com/jkortus/alg.cubing.net.

**Comment by DougCube** [comment]
Ah, so the solution is to fork my own version if I want my own color scheme. Got it.
[dougcube.alg.cubing.net](https://dougcube-alg-cubing-net.onrender.com/)

**Comment by lgarron** [comment]
Yeah, I think at this point I would encourage using/hosting your own fork if you need a different color scheme.

See https://github.com/cubing/cubing.js/issues/157#issuecomment-2641779745 for more info.
